.travis.yml

```diff
@@ -3,42 +3,49 @@ services: docker
 language: bash
```

Comment:
You changed your new modification with 2 spaces indent. But this `.travis.yml` YAMP file was managed with 4 spaces indent. Please change your modification in `.travis.yml` to 4 spaces.

Reply:
> I think we do not include `TRAVIS_TAG` and `TRAVIS_REPO_SLUG` variables in '*.sh".
> Because we (at least me) want to run the shell scripts on my local environment.
> 
> If you keep below variables, it is possible not to use the `TRAVIS-*` in `.*.sh`.
> 
> ```
> -    global:
> -        - VERSION=4.0.0-5
> -        # See qemu-user-static's RPM spec file on Fedora to check the new version.
> -        # https://src.fedoraproject.org/rpms/qemu/blob/master/f/qemu.spec
> -        - DOCKER_SERVER=docker.io
> -        # - DOCKER_SERVER=quay.io
> -        # Git repository
> -        - REPO=multiarch/qemu-user-static
> -        # Container repository
> -        - DOCKER_REPO=$DOCKER_SERVER/multiarch/qemu-user-static
> -        # - DOCKER_REPO=$DOCKER_SERVER/your_username/qemu-user-static
> -        - PACKAGE_URI="https://kojipkgs.fedoraproject.org/packages/qemu/4.0.0/5.fc31/x86_64/q
> emu-user-static-4.0.0-5.fc31.x86_64.rpm"
> -        - PACKAGE_FILENAME=$(basename "$PACKAGE_URI")
> ```
> 
> Keep `docker images` that is after building containers on local.
> That's useful to check actual created containers with tags.

If you run the script with:

```bash
TRAVIS_REPO_SLUG=multiarch/qemu-user-static ./test.sh
```

it should work, that being said I'm happy to add fallback to all `TRAVIS_` vars in `.sh`: `${TRAVIS_REPO_SLUG:-$REPO}`, or hardcoded `${TRAVIS_REPO_SLUG:-multiarch/qemu-user-static}`.  Main reason I've added `TRAVIS_` vars there is because I couldn't get `.travis.yml`-defined global vars to be available in scripts.